apps/extension/src/Approvals/Commitment.tsx

```diff
@@ -17,15 +17,12 @@ import { FaVoteYea } from "react-icons/fa";
 import { FaRegEye, FaWallet } from "react-icons/fa6";
 import { GoStack } from "react-icons/go";
 import { PiDotsNineBold } from "react-icons/pi";
-import {
-  hasShieldedSection,
-  parseTransferType,
-  ShieldedPoolLabel,
-} from "utils";
+import { isShieldedPool, parseTransferType, ShieldedPoolLabel } from "utils";
 import { TransactionCard } from "./TransactionCard";
 
 type CommitmentProps = {
   commitment: CommitmentDetailProps;
+  wrapperFeePayer?: string;
```

Comment:
Question: Wouldn't `wrapperFeePayer` always be defined here? Even if we don't provide it as an initial Tx argument, I think this will always be set to an address by the time it is deserialized. If that's the case, this wouldn't need to be optional, but I'm not completely sure

Reply:
Yeah I think it's always there. I can typecast :D